ui/components-lib/src/components/Card/_layouts.scss

```
@@ -48,27 +58,45 @@ $extension-wrapper-gap: 20px;
-    min-height: 400px;
     display: flex;
     flex-direction: column;
+    gap: 0;
+  }
 
-    &.single-commit-group {
-      min-height: 250px;
-    }
+  .active-card {
+    @include card-chrome;
+    display: flex;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)